### DISCUSSION TITLE
Command failed: git --no-pager --git-dir

### DISCUSSION THREAD

**Discussion by GoldenSheep402** [rust-lang/crates.io#5582] [Q&A]
![image](https://user-images.githubusercontent.com/67376942/205448212-72808113-c363-4fee-8825-6d60300ea1c0.png)
I tried to change my proxy and everything I could think about. This error never changed.
Is there anyone who know why?

**Comment by rhettli** [comment]
i got the same problem like you

**Reply by GoldenSheep402** [reply]
it seems the version of the plugin problem. Maybe I forgot that,you chould try

**Reply by GoldenSheep402** [reply]
it seems the version of the plugin problem. Maybe I forgot that,you chould try
